admin_app/src/components/ContentCard.tsx

```diff
@@ -2,10 +2,29 @@
 import React, { useState } from "react";
 import { PencilIcon, TrashIcon } from "@heroicons/react/20/solid";
 
-export type Content = {
-  content_id: string;
+export enum Language {
+  ENGLISH = "ENGLISH",
+  HINDI = "HINDI",
+}
+
+export type ContentDataInEdit = {
+  content_title: string | null;
+  content_text: string | null;
+  content_language: Language | null;
+};
+
+export type ContentData = {
   content_title: string;
   content_text: string;
+  content_language: Language;
+};
```

Comment:
`ContentDataInEdit` can have null values, since it could be a new content being created.

However whenever we save or retrieve or delete a card, we expect none of these fields to be null, so we shall use `ContentData` type.

Reply:
I also created these data types so we can reuse them in page.tsx

Reply:
Thanks! We should link these types at some stage `ContentDataInEdit` is a superset of `ContentData`. But we can do that another day / another PR